digital-twin-model/dtdl/v3/content/obd.json

```diff
@@ -0,0 +1,17 @@
+[
```

Comment:
Is this the same as the OBD DTDL used in other samples? If so, can we consolidate them so that there's less duplication?